instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Comment from './Comment';
+import './CommentSection.css';
+
+CommentSection.propTypes = {
```

Comment:
You'll need to import PropTypes into the file to use them.

Reply:
Once you import PropTypes, you'll get a warning that CommentSection is being used before it's called. You can move your propTypes object tot he bottom of the file to silence that warning.